dhis-2/dhis-services/dhis-service-reporting/src/main/java/org/hisp/dhis/interpretation/impl/DefaultInterpretationService.java

```diff
@@ -208,47 +240,47 @@ public long getNewInterpretationCount()
 
         return count;
     }
-    
+
```

Comment:
Those spaces are messing the diff a bit.

Reply:
Yes, it is a bit of a pain because I am using the Dhis2 Formatter for Eclipse but still some files are not properly formatted so if I apply the format I screw everything up. Anything I have at least reverted these ones...